cpp/hello.cpp

```diff
@@ -0,0 +1,6 @@
+#include <iostream>
+
+int main() {
+    std::cout << "Hello, World!" << std::endl;
```

Comment:
Ahh! Missed this in my review (I added tests and it caught it). Should be lowercase, no punctuation:

```suggestion
    std::cout << "hello, world" << std::endl;
```

Reply:
It has been done. "Hello, World!" >> "hello, world"